### PR TITLE
add axiosConfig for provider

### PR DESCRIPTION
**Pull request by shyun-comcom**
Add `axiosConfig` for provider setting to resolve `(e.adapter || s.adapter) is not a function` issue in wallet extension.

### CODE REVIEW

**Comment by platfowner** [comment]
@shyun-comcom could you check the test pass?

File: src/types.ts [hunk 1]
```diff
@@ -48,6 +48,7 @@ export interface V3Keystore {
 
 export type AinOptions = {
   rawResultMode?: boolean;
+  axiosConfig?: any;
```
Comment (ehgmsdk20) [inline src/types.ts:51]: How about using `AxiosRequestConfig` in axios library?

**Review by ehgmsdk20 — APPROVED** [review]
LGTM with a suggestion, thanks!

**Review by platfowner — APPROVED** [review]
LGTM
Thanks!

File: src/types.ts (hunk 1 above)
Comment (shyun-comcom) [reply src/types.ts]: Fixed

File: src/ain.ts [hunk 2]
```diff
@@ -43,7 +47,7 @@ export default class Ain {
    * @param {string} providerUrl
    */
   setProvider(providerUrl: string, chainId?: number) {
```
Comment (ehgmsdk20) [inline src/ain.ts:49]: how about getting new AxiosConfig when setting a new provider in optional?

Comment (shyun-comcom) [reply src/ain.ts]: Fixed!

**Comment by ehgmsdk20** [comment]
LGTM again, thanks!

**Pull request merged**